### PR TITLE
SER-728 Create Verve Card Type. 

### PR DESCRIPTION
**Pull request by jherreraa**
This change add new credit card brand Verve and tests related to it.

### CODE REVIEW

**Review by sinourain — APPROVED** [review]
LGTM

**Review by Heavyblade — COMMENTED** [review]
the basic implementation looks 👍 @jherreraa, lef a style comment and I have one question:

❓ => Shouldn't this PR also include the addition of the new credit card type to Rapyd?

File: test/unit/credit_card_methods_test.rb [hunk 1]
```diff
@@ -506,6 +506,38 @@ def test_should_detect_panal_card
...
+    end
+
+    assert_equal 19, verve.length
+
+    verve << '0'
+    assert_not_equal 'verve', CreditCard.brand?(verve)
+  end
+
+  def test_should_detect_verve
+    assert_equal 'verve', CreditCard.brand?('5060990000000000')
```
Comment (Heavyblade) [inline test/unit/credit_card_methods_test.rb:527]: 💬 Comment Style:

maybe an array of cc numbers + an each to check?

```ruby
  def test_should_detect_verve
    credit_cardsdit_cards = %w[5060990000000000
             506112100000000000
             5061351000000000000
             5061591000000000
             506175100000000000
             5078801000000000000
             5079381000000000
             637058100000000000
             5079400000000000000
             507879000000000000
             5061930000000000
             506136000000000000]
    credit_cards.all? { |cc| CreditCard.brand?(cc) == 'verve' }
  end

```

File: test/unit/credit_card_methods_test.rb [hunk 2]
```diff
@@ -506,6 +506,38 @@ def test_should_detect_panal_card
...
+    verve = '506099000000000'
+
+    assert_equal 15, verve.length
+    assert_not_equal 'verve', CreditCard.brand?(verve)
+
+    while verve.length < 19
+      verve << '0'
+      assert_equal 'verve', CreditCard.brand?(verve), "Failed for bin #{verve}"
+    end
+
```
[the remark this answers is not in the document]
Comment (jherreraa) [reply test/unit/credit_card_methods_test.rb:519]: Corrected! 👍

File: test/unit/credit_card_methods_test.rb (hunk 1 above)
Comment (jherreraa) [reply test/unit/credit_card_methods_test.rb:527]: Corrected! 👍

**Review by naashton — APPROVED** [review]
This looks good! Can we also include a `CHANGELOG` entry?

**Pull request merged**